### ISSUE TITLE
Delay land acquisition until army returns.

### ISSUE THREAD

**Issue by klondikemarlen** [elthran/UndyingKingdoms#107]
I suggest delaying the acquisition of new land until the conquering army returns. This will prevent the he who attacks first loses problem. Said problem arises due to the mathematics of land acquisition being employed.

e.g.
Player 1 - 150 land, player 2 - 150 land
Player 1 attacks player 2 and steals 12% (18) of their land.
Now player 1 has 168 land.
Player 2 attacks player 1 and steals 12% (20) of their land.
Now player 2 has 152 land.
Player 1 attacks player 2 and steals 12% (18) of their land.
Now player 1 has 166 land.
Player 2 attacks player 1 and steals 12% (20) of their land.
Now player 2 has 154 land.

I'm not even sure now that this won't simply level out. More analysis is required.

**Comment by elthran** [comment]
Yes, I agree. I want the results to be instantly shown to you, but your land should be come available when your army returns. Also, the 12% land taken per attack is a placeholder. In the future it will be a complicated formula based on relative land sizes and how big your army is.
